bindings/nodejs/npm/darwin-x64/README.md

```diff
@@ -0,0 +1,3 @@
+# `@glaredb/glaredb-darwin-x64`
+
+This is the **x86_64-apple-darwin** binary for `@glaredb/glaredb`
```

Comment:
Are we able to get a quick example in the README?

I think the pypi one is good enough to duplicate/go off of: https://pypi.org/project/glaredb/

Particularly:

- link to repo
- example of how to use it

Reply:
the `/npm/*/README` files are generated from napi. Users should never interact with these directly. 

Ill add a readme to the js bindings with a quick helloworld.

Reply:
sweet!